tests/components/tts/common.py

```diff
@@ -263,3 +265,26 @@ async def async_setup_entry_platform(
     await hass.async_block_till_done()
 
     return config_entry
+
+
+class MockResultStream(ResultStream):
```

Comment:
Are we going to use this in other test than the one added here? Why don't we simply patch the tts manager methods involved in the `ResultStream` methods and instantiate a result stream in the test?

Reply:
Yep, it's going to be used in https://github.com/home-assistant/core/pull/139671, https://github.com/home-assistant/core/pull/139668, https://github.com/home-assistant/core/pull/139665, https://github.com/home-assistant/core/pull/140336 and the subsequent migrations of announce calls to tokens.

Reply:
Ok.